CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 
 * Adds the following new processor to `Minfraud::Components::Payment`:
   * `:windcave`
+* The `last_4_digits` input to `Minfraud::Components::CreditCard` has been
+  deprecated in favor of `last_digits` and will be removed in a future
+  release. `last_digits`/`last_4_digits` also now supports two digit values
+  in addition to the previous four digit values.
+* Eight digit `issuer_id_numbers` inputs are now supported by
+  `Minfraud::Components::CreditCard` in addition to the previously accepted
+  six digit `issuer_id_number`s. If you send six digits for the
+  `issuer_id_number`, you should send the last four digits for `last_digits`.
+  If you send eight digits for `issuer_id_number`, you should send the last
+  two digits for `last_digits`.
```

Comment:
I noticed discussion on another related PR regarding a similar description. Perhaps we should get the same description in all spots (or link to somewhere with guidance)?